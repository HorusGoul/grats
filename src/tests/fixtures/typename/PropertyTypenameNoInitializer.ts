/** @GQLType */
export class User {
  __typename: string;
  /** @GQLField */
  name: string = "Alice";
}
