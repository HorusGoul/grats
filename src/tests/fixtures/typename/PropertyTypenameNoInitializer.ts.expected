-----------------
INPUT
----------------- 
/** @GQLType */
export class User {
  __typename: string;
  /** @GQLField */
  name: string = "Alice";
}

-----------------
OUTPUT
-----------------
src/tests/fixtures/typename/PropertyTypenameNoInitializer.ts:3:3 - error: Expected `__typename` property to have an initializer. For example: `__typename = "MyType"`.

3   __typename: string;
    ~~~~~~~~~~
